Makefile

```diff
@@ -15,9 +15,14 @@ GIT_REVISION := $(shell git rev-parse HEAD)
 # This is analogous to revisions in DEB and RPM archives.
 revision = $(if $(REVISION),$(REVISION),)
 
-.PHONY: all build lint container container-oss config-svc-build config-svc-container config-svc-test-unit container-public container-lint container-scan dist test-dist container-clean clean examples example-containers test test-kubernetes test-native test-containers test-unit docs-license-analysis
+.PHONY: all build clean config-svc-lint container container-clean container-lint container-oss \
+		container-public container-scan dist docs docs-generate-markdown docs-license-analysis \
+		docs-lint example-kubernetes example-containers example-multi example-kubernetes-oss \
+		example-containers-oss example-multi-oss lint test test-containers test-dist test-kubernetes \
+		test-native
```

Comment:
bit of a weird split, might be better to keep `test-` together, etc.

Reply:
Yeah just did it alphabetically but the linebreaks could use some work

Reply:
Yeah probably easier to keep things together by "function" then most generic first before rest by alphabetical

`test test-containers test-dist test-native`

Reply:
Very much a nit pick though